Assets/Prefabs/Player.prefab

```diff
@@ -2323,7 +2323,7 @@ GameObject:
   - component: {fileID: 6709035963052178592}
   m_Layer: 0
   m_Name: Colliders
-  m_TagString: Untagged
+  m_TagString: PlayerCollider
```

Comment:
You have added a tag `PlayerCollider` that from what i can see you're not using. Remove the tag and set the player's colliders back to untagged.